src/processing/resources/page.js

```diff
@@ -22,6 +21,8 @@ class PageProcessor extends ResourceProcessorBase {
         this.domProcessor = new DomProcessor(new DomAdapter());
     }
 
+    restartProcessing = Symbol();
```

Comment:
We don't use class properties anymore because more likely they will not be in the spec (see #29), so move it to ctor, use const notation and `new` keyword:
```js
this.RESTART_PROCESSING = new Symbol();
```